### ISSUE TITLE
Fix <a> tag nesting error

### ISSUE THREAD

**Issue by kestrelgorlick** [Binge-Prime/Binger#9]
The AllProducts component is throwing an error in the browser console:
<img width="329" alt="Screen Shot 2021-02-13 at 4 15 17 PM" src="https://user-images.githubusercontent.com/6307252/107864239-60119400-6e17-11eb-9c92-906d5225e965.png">

I identified the source, we have nested links in our products:
<img width="1226" alt="Screen Shot 2021-02-13 at 4 15 41 PM" src="https://user-images.githubusercontent.com/6307252/107864261-a8c94d00-6e17-11eb-98ec-b4a06a21da53.png">

Basically, when webpack builds this, we are ending up with links within links, i.e. `<a><a></a></a>`, which is not good. 

I tinkered with this for a bit, and I think the best solution is to contain the parent `<Link>` (line 23) in a parent `<div>` and then move the logic-based return (lines 33-44) outside the parent `<Link>`. This will make our `<a>` tags un-nested. It should be a really easy and quick fix.

Cheers!
